apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -553,6 +558,8 @@ const EventTypePage = (props: EventTypeSetupProps & { allActiveWorkflows?: Workf
     const updatedFields: Partial<FormValues> = {};
     Object.keys(dirtyFields).forEach((key) => {
       const typedKey = key as keyof typeof dirtyFields;
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
       updatedFields[typedKey] = undefined;
```

Comment:
For some reason getting this type error on this line, and at all not able to fix this, therefore added this. If you have some solution please let me know. 🙏 
FYI - few lines below if you see [for this line](https://github.com/calcom/cal.com/pull/15896/files#diff-df6d4a41111a432e757de2e1297681f80d9a8d9d74641c89e8f1a12333d4a09aR568) the same `ts-ignore` is added, maybe it was also getting the same error.

`Type error: Expression produces a union type that is too complex to represent.`